src/promptflow/promptflow/_core/openai_injector.py

```diff
@@ -133,11 +226,10 @@ def inject_openai_api():
     It stores the original methods as _original attributes of the create methods.
     2. Updates the openai api configs from environment variables.
     """
-    for openai_api in available_openai_apis():
-        # Check if the create method of the openai_api class has already been modified
```

Comment:
Please keep this line of comment as well.

Reply:
OK